examples/seq2seq/seq2seq.py

```diff
@@ -26,6 +26,23 @@ def sequence_embed(embed, xs):
     return exs
 
 
+def get_topk(x, k=5, axis=1):
+    ids_list = []
+    scores_list = []
+    xp = cuda.get_array_module(x)
+    for i in range(k):
+        ids = xp.argmax(x, axis=axis).astype('i')
```

Comment:
In my environment (chainer 1.21.0), `cupy.argmax` exists. It was implemented before I knew.